lib/ht/search_client/test.rb

```diff
@@ -4,6 +4,10 @@ module Ht::SearchClient::Test
     def stub_request(params = {})
       new(params).stub_request
     end
+
+    def stub_post_request(params = {})
```

Comment:
I tried several approaches before duplicating the whole stuff but none was working as I wanted it too :cry: 
Basically the problem is that the actual stubbing is buried under 5 (five) levels of indirection in the same class, and the parameter I was passing down the line (`method: :post`) is rejected in the `allowed_params` calls. I tried to refine that but no luck there.